### ISSUE TITLE
Add installation of PHPMyAdmin at the end

### ISSUE THREAD

**Issue by danimalweb** [creativedotdesign/vesta-setup#2]
Hey @kronflux. Thanks for the updates I found in your fork.

I wondered if you had the commands for reinstalling PHPMyAdmin after removing it.

Cheers.

**Comment by kronflux** [comment]
There's really nothing to installing PHPMyAdmin, it's just `apt-get install phpmyadmin`
Unfortunately I didn't get to use my updated guide much. I was preparing it for my own use(so I could repeat my own steps in the future, more easily for my specific setup) but during my updating this, I was using a virtual machine - in practice, I was hoping to run it on a Raspberry Pi, and didn't realize at the time that Vesta doesn't support ARM.

I did request they support ARM, and it was addressed as a feature request, so hopefully in the future they will. If/when that happens, I'll completely update this guide for use on a raspberry pi, and even update it with instructions for PHP 7.1, rather than 7.0.

The original intent was to have a full guide to not only set up Vesta, but also a full guide on installing NextCloud within Vesta with this setup. So keep an eye out on my fork :)

Cheers!
